src/main/scala/mesosphere/marathon/api/akkahttp/V2Controller.scala

```diff
@@ -2,7 +2,7 @@ package mesosphere.marathon
 package api.akkahttp
 
 import akka.http.scaladsl.server.Route
-import v2.{ AppsController, EventsController, InfoController, PluginsController }
+import v2._
```

Comment:
What's the benefit of this? Does it import some implicits? Otherwise I'd keep the explicit imports. This makes reading the code much easier.

Reply:
Intellij is set to to automatically convert >= 5 explicit imports to `_`. I don't see any particular benefit in changing this - long import lists aren't exactly handy. And look what I found in the leader controller PR:
https://github.com/mesosphere/marathon/pull/5579/files#diff-9dab674675063e0e5738657574bd846eR5
;)

Reply:
Damn it. Ok. If it's part of the style...